### PR TITLE
Update CONTRIBUTING.md with instructions on how to load unpackaged build into Chrome

### PR DESCRIPTION
**Pull request by BaruYogesh**
[discord](https://discord.com/channels/617136488840429598/1081538711742844980/1260800390434132039)
Screenshot 2024-07-10 at 11 50 20 PM

### CODE REVIEW

**Comment by Kuuuube** [comment]
This looks good but is there any reason you added a height to your image but not a width? I assume adding height is to prevent reflowing which is fine but it distorts the image if you dont also add width.

**Comment by BaruYogesh** [comment]
> This looks good but is there any reason you added a height to your image but not a width? I assume adding height is to prevent reflowing which is fine but it distorts the image if you dont also add width.

nope, just wasn't aware. i have updated the dimensions to be half that of the original, otherwise the image is overwhelmingly large.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -57,6 +57,16 @@ Several command line arguments are available for these scripts:
 
 If no arguments are specified, the command is equivalent to `build.bat --all`.
 
+### Loading unpacked build into chrome
```
Comment (Kuuuube) [inline CONTRIBUTING.md:60]:
```suggestion
### Loading unpacked build into Chrome
```

**Review by Kuuuube — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -57,6 +57,19 @@ Several command line arguments are available for these scripts:
 
 If no arguments are specified, the command is equivalent to `build.bat --all`.
 
+### Loading unpacked build into Chrome
```
Comment (MarvNC) [inline CONTRIBUTING.md:60]:
```suggestion
### Loading unpacked build into Chromium browsers
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -57,6 +57,19 @@ Several command line arguments are available for these scripts:
...
+
+After building, you can load the compiled extension into Chrome
```
Comment (MarvNC) [inline CONTRIBUTING.md:62]:
```suggestion
After building, you can load the compiled extension into Chromium browsers.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -57,6 +57,19 @@ Several command line arguments are available for these scripts:
...
+
+- Navigate to [extensions settings](chrome://extensions/)
```
Comment (MarvNC) [inline CONTRIBUTING.md:64]:
```suggestion
- Navigate to the [extensions page](chrome://extensions/)
```

**Review by MarvNC — APPROVED** [review]

**Review by jamesmaa — APPROVED** [review]

**Review by cashewnuttynuts — APPROVED** [review]

**Pull request merged**